### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by lucasmz-dev**

### CODE REVIEW

**Comment by lucasmz-dev** [comment]
I see some references to the ability to restore data with Seedvault after setup. I'm not sure if this has been merged or not and so I haven't included it. Let me know if I should.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+## [14-5.0] - 2024-08-22
+* It is now possible to select what to restore (e.g. apps, files...)
+* Now in Material 3
+* Automatic backup scheduling can now be modified by the user
+* Native support for WebDAV mounts
+* Support for RoundSync
```
Comment (grote) [inline CHANGELOG.md]: (if enabled by your ROM...)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+* Apps which APKs failed to backup are now considered failed
```
Comment (grote) [inline CHANGELOG.md]: Not sure what this refers to, could probably get removed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+* The name of the profile is now shown when selecting a backup to restore
+* Already installed apps are not reinstalled anymore
+* Already stored files are not overwritten anymore
```
Comment (grote) [inline CHANGELOG.md]: they weren't overriden, but duplicates created. no more duplicates, if files had no changes.

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -7,8 +7,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     package="com.stevesoltys.seedvault"
-    android:versionCode="34040000"
-    android:versionName="14-4.0">
+    android:versionCode="34050000"
+    android:versionName="14-5.0">
```
Comment (grote) [inline app/src/main/AndroidManifest.xml]: I am personally a friend of smaller bumps, like 4.1 now.

**Review by grote — COMMENTED** [review]
Awesome. Thanks a lot!

> I see some references to the ability to restore data with Seedvault after setup. I'm not sure if this has been merged or not and so I haven't included it. Let me know if I should.

This should go in soon. Once https://github.com/seedvault-app/seedvault/pull/723 is merged. Could already be added to the changelog and we merge this PR afterwards!

cd @chirayudesai

**Comment by lucasmz-dev** [comment]
> This should go in soon. Once https://github.com/seedvault-app/seedvault/pull/723 is merged. Could already be added to the changelog and we merge this PR afterwards!

My only adversion to this is that it might look weird in git history. The true start of 14-4.1 would be when the merge commit shows up, but the commit for the bump and changelog itself may come before that, which may cause confusion.

Also do you guys not do tags? I see only for like 2021 versions, it makes me a bit confused on whether Seedvault is meant to be merged constantly or based on versions

**Review by t-m-w — APPROVED** [review]
Looks good to me! Version is updated as expected when building with this, in app info and About page.

So I guess the remaining question is when to merge? And then if there are more changes merged before then, update the changelog again?

**Pull request merged**